lib/lambda_vault_auth.rb

```diff
@@ -17,6 +14,8 @@ class Vaulter
                 :renewal_window,
                 :ttl
 
+    DEFAULT_STS_URI = 'https://sts.amazonaws.com'.freeze
+
     def initialize(sts = Aws::STS::Client.new, env = ENV)
```

Comment:
Let's remove the STS client now that it isn't directly necessary

Reply:
What the github review is terrible? 

L#84 can go.

Reply:
OK, removed the STS client refs.